libs/partners/anthropic/langchain_anthropic/output_parsers.py

```diff
@@ -33,11 +26,16 @@ def parse_result(self, result: List[Generation], *, partial: bool = False) -> An
         """
         if not result or not isinstance(result[0], ChatGeneration):
             return None if self.first_tool_only else []
-        tool_calls: List = _extract_tool_calls(result[0].message)
+        message = result[0].message
+        if isinstance(message.content, str):
+            tool_calls: List = []
+        else:
+            content: List = message.content
+            tool_calls = extract_tool_calls(content)
```

Comment:
should we convert these to dicts (instead of pydantic objects). would better match our other tool parsers and pydantic objects seem overkill here. also to avoid a breaking change

Reply:
fixed this, thanks